lecture3/main.c

```diff
@@ -0,0 +1,193 @@
+#include "task3.c"
+
+int main(void)
+{
+	char istring[101], ostring[100];
+	int digit;
+
+	printf("   Exercise 1\n");
+	printf(" Input string (100 characters maximum!):\n");
+	gets(istring);
+	printf(" Output string:\n");
+	copy_string(istring,ostring);
+	printf("%s\n",ostring);
+
+	printf("   Exercise 2\n");
+	printf(" Input 3 digits to push in stack:\n");
+	printf("input first digit: ");scanf("%d",&digit);
+	if(Push(digit) == 1)
+	{
+		printf("  First digit is in stack!\n");
+	}
+	else
+	{
+		printf("  Cannot push first digit in stack!\n");
+	}
+	printf("input second digit: ");scanf("%d",&digit);
+	if(Push(digit) == 1)
+	{
+		printf("  Second digit is in stack!\n");
+	}
+	else
+	{
+		printf("  Cannot push second digit in stack!\n");
+	}
+	printf("input third digit: ");scanf("%d",&digit);
+	if(Push(digit) == 1)
+	{
+		printf("  Third digit is in stack!\n");
+	}
+	else
+	{
+		printf("  Cannot push third digit in stack!\n");
+	}
+
+
+	if(Pop(&digit) == 1)
+	{
+		printf("  Last digit retrieved from stack is  %d\n",digit);
+	}
+	else
+	{
+		printf("  Cannot pop digit from stack!\n");
+	}
+	if(Pop(&digit) == 1)
+	{
+		printf("  Last digit retrieved from stack is  %d\n",digit);
+	}
+	else
+	{
+		printf("  Cannot pop digit from stack!\n");
+	}
+	if(Read(&digit) == 1)
+	{
+		printf("  Last digit in stack is  %d\n",digit);
+	}
+	else
+	{
+		printf("  Cannot read digit from stack!\n");
+	}
+	if(Pop(&digit) == 1)
+	{
+		printf("  Last digit retrieved from stack is  %d\n",digit);
+	}
+	else
+	{
+		printf("  Cannot pop digit from stack!\n");
+	}
+	if(Read(&digit) == 1)
+	{
+		printf("  Last digit in stack is  %d\n",digit);
+	}
+	else
+	{
+		printf("  Cannot read digit from stack!\n");
+	}
+	if(Pop(&digit) == 1)
+	{
+		printf("  Last digit retrieved from stack is  %d\n",digit);
+	}
+	else
+	{
+		printf("  Cannot pop digit from stack!\n");
+	}
+	if(Read(&digit) == 1)
+	{
+		printf("  Last digit in stack is  %d\n",digit);
+	}
+	else
+	{
+		printf("  Cannot read digit from stack!\n");
+	}
+
+	printf("   Exercise 3\n");
+	if(GetQueue(&digit) == 1){printf("  Get %d from queue\n",digit);}
```

Comment:
Code style is inconsistent, which makes it hard to review. A few lines above you've added newlines after each else and each bracket, and here you don't add it.